android/gradlew.bat

```diff
@@ -40,7 +40,7 @@ if defined JAVA_HOME goto findJavaFromJavaHome
 
 set JAVA_EXE=java.exe
 %JAVA_EXE% -version >NUL 2>&1
-if "%ERRORLEVEL%" == "0" goto init
+if "%ERRORLEVEL%" == "0" goto execute
```

Comment:
Did your changes in `android/gradlew.bat` occur here as a result of building with a new version of Gradle in AndroidStudio?

If not, could you link me to the documentation you followed for this?

Reply:
Correct, I did not make any direct changes to `android/gradlew.bat`.  These changes probably came from a new version of Gradle.